packages/flutter_tools/lib/src/template.dart

```diff
@@ -122,6 +124,8 @@ class Template {
 
       finalDestinationFile.writeAsBytesSync(sourceFile.readAsBytesSync());
     });
+
+    printStatus('$createdFileCount files created.');
```

Comment:
I reverted this - it is nice to let the user know what files are in the project. I did simplify the output a bit.